ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datetime/v1/datetime/README.md

```diff
@@ -0,0 +1,92 @@
+* Styles
+* Allows replacing this component with other component (as mentioned below).
+
+### Use Object
+The Form Date & Time component uses the `com.adobe.cq.forms.core.components.models.form.Datetime` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Form Date & Time component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this field
+2. `./hideTitle` - if set to `true`, the label of this field will be hidden
+3. `./name` - defines the name of the field, which will be submitted with the form data
+4. `./description` - defines a help message that can be rendered in the field as a hint for the user
+5. `./required` - if set to `true`, this field will be marked as required, not allowing the form to be submitted until the field has a value
+6. `./requiredMessage` - defines the message displayed as tooltip when submitting the form if the value is left empty
+7. `./readOnly` - if set to `true`, the filed will be read only
+8. './minimumDateTime' - defines the minimum date and time that can be selected
+9. './maximumDateTime' - defines the maximum date and time that can be selected
+10. './minimumMessage' - defines the minimum date and time message that will come when the selected date and time is before the minimum date and time
+11. './maximumMessage' - defines the maximum date and time message that will come when the selected date and time is after the maximum date and time
+12. './default' - defines the default date and time that will be selected when the form is loaded
+
+## Client Libraries
+The component provides a `core.forms.components.datetime.v1.runtime` client library category that contains the Javascript runtime for the component.
+It should be added to a relevant site client library using the `embed` property.
+
+It also provides a `core.forms.components.datetime.v1.editor` editor client library category that includes
+JavaScript handling for dialog interaction. It is already included by its edit dialog.
+
+## BEM Description
+```
+BLOCK cmp-adaptiveform-datetime
+    ELEMENT cmp-adaptiveform-datetime__label
+    ELEMENT cmp-adaptiveform-datetime__label-container
+    ELEMENT cmp-adaptiveform-datetime__widget
+    ELEMENT cmp-adaptiveform-datetime__questionmark
+    ELEMENT cmp-adaptiveform-datetime__shortdescription
+    ELEMENT cmp-adaptiveform-datetime__longdescription
+    ELEMENT cmp-adaptiveform-datetime__errormessage
+```
+
+### Note
+By placing the class names `cmp-adaptiveform-datetime__label` and `cmp-adaptiveform-datetime__questionmark` within the `cmp-adaptiveform-datetime__label-container` class, you create a logical grouping of the label and question mark elements. This approach simplifies the process of maintaining a consistent styling for both elements.
+
+## Replace feature:
+We support replace feature that allows replacing Reset Button component to any of the below components:
```

Comment:
I was referring to the fact that the documentation mentions a Reset button, but the component in question is a DateTime field. Please review the entire PR holistically.

Reply:
fixed it